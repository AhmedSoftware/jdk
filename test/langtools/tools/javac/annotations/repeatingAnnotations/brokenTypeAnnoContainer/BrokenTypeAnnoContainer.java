/**
 * @test    /nodynamiccopyright/
 * @bug     8044196
 * @summary Ensure that a broken type annotation container generates a correct error message.
 * @compile T.java TC.java
 * @compile -J-XX:+UnlockExperimentalVMOptions -J-XX:hashCode=2 T.java TC.java
 * @compile TCBroken.java
 * @compile -J-XX:+UnlockExperimentalVMOptions -J-XX:hashCode=2 TCBroken.java
 * @compile/fail/ref=BrokenTypeAnnoContainer.out -XDrawDiagnostics BrokenTypeAnnoContainer.java
 */

class BrokenTypeAnnoContainer {
    void method() {
        int ll2 = (@T(1) @T(2) int) 0;
    }
}
