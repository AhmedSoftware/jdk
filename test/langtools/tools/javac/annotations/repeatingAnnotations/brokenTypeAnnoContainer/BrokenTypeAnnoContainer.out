BrokenTypeAnnoContainer.java:14:20: compiler.err.duplicate.annotation.invalid.repeated: T
1 error
